lib/capybara/assertions.rb

```diff
@@ -290,5 +290,37 @@ def prepare_args(args)
         [page, *args]
       end
     end
+
+    module CapybaraFailureHelpers
+      def self.failure_message(description, options={})
+        String.new("expected to find #{description}") << count_message(options)
+      end
+
+      def self.negative_failure_message(description, options={})
+        String.new("expected not to find #{description}") << count_message(options)
+      end
+
+      def self.declension(singular, plural, count)
+        if count == 1
+          singular
+        else
+          plural
+        end
+      end
+
+      def self.count_message(options)
+        message = String.new()
+        if options[:count]
```

Comment:
since all of the branches are mutually exclusive, there's no need to mutate the `message` string. Could you rewrite to:

```ruby
def self.count_message(options)
  case
  when options[:count]
    " #{options[:count]} #{declension('time', 'times', options[:count])}"
  when ...
    ...
  end
end
```